### PR TITLE
2020-01-05

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2020-01-06-programming-tutorials.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+type: post
+published: true
+status: publish
+categories: [Programming]
+image: /assets/article_images/roman-mager-59976.jpeg
+image2: /assets/article_images/roman-mager-59976.jpeg
+author: Valentino Urbano
+---
+
+Everyone has read a tutorial on a programming blog somewhere on some topic they were interested in, but few programmers have a blog and even fewer have ever written a tutorial on it. I've written before at length about why you should have a personal blog so I won't tackle that topic here, but only focus on why you should start thinking about writing tutorials.
```
Comment (MsSlacker) [inline _posts/2020-01-06-programming-tutorials.md]: This whole paragraph gave me a headache... Sentences here are too long and confusing.

File: _posts/2020-01-06-programming-tutorials.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+I have written a few small ones and with each and every one of them, even without having to go into too much depth since they were not very long, I have learned more than I could ever do otherwise on that specifing topic.
+
+The reason is that **before teach others, you need to have understood the topic in its entirety yourself**. You can't get away with skimming some documentation and only understand superficially. It is not going to cut it.
```
Comment (MsSlacker) [inline _posts/2020-01-06-programming-tutorials.md]: before you teach?

File: _posts/2020-01-06-programming-tutorials.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Of course this doesn't only apply to programmers[^1], but to any kind of content creation in general. And you? What are you going to write about?
```
Comment (MsSlacker) [inline _posts/2020-01-06-programming-tutorials.md:20]: And you (...) seems way out of place...

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2020-01-06-programming-tutorials.md (hunk 1 above)
Comment (valeIT) [reply _posts/2020-01-06-programming-tutorials.md]: Better or still off?

File: _posts/2020-01-06-programming-tutorials.md (hunk 3 above)
Comment (valeIT) [reply _posts/2020-01-06-programming-tutorials.md:20]: mm better?

**Review by MsSlacker — APPROVED** [review]
Now that's waaaay better

**Pull request merged**